src/core/File.ts

```diff
@@ -221,6 +221,7 @@ export class File {
      * @memberOf File
      */
     public tryPlugins(_ast?: any) {
+        if (this.context.runAllMatchedPlugins) { this.tryAllPlugins(_ast) }
```

Comment:
I think in this case you will have them both run (allPlugins + the current scenario). I don't think it's correct